docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -160,26 +160,6 @@ Copy the machine account public key somewhere safe. You will need it in a later
 
 </Callout>
 
-### Upload your Public Keys
```

Comment:
Essentially removing this section https://docs.onflow.org/node-operation/node-bootstrap/#upload-your-public-keys

Reply:
actually thats a good catch..we do need them to run this is they are running a consensus node. Will fix.